pyproject.toml

```
@@ -58,9 +58,10 @@ typing-extensions = "*"
 msgspec = ">=0.11.0"
 jinja2 = { version = ">=3.1.2", optional = true }
 mako = { version = ">=1.2.4", optional = true }
-click = {version = ">=8.1.3", optional = true}
-rich = {version = ">=13.0.1", optional = true}
-importlib-metadata = { version = ">=5.2.0", python = "<3.10" }
+click = {version = "*", optional = true}
+rich = {version = "*", optional = true}
```

```suggestion
rich = {version = ">=13.0.0", optional = true}
```

This is required. Can't pinpoint what it was, but something with the tables caused issues in earlier versions